test/gui/tst_moveFilesFolders/test.feature

```diff
@@ -65,3 +65,12 @@ Feature: move file and folder
         And as "Alice" the file "folder1/folder2/file2.txt" should have the content "test file 2" in the server
         And as "Alice" file "folder1/file1.txt" should not exist in the server
         And as "Alice" file "folder1/file2.txt" should not exist in the server
+
+    @skipOnWindows @issue-435
+	Scenario: Syncing a 50MB file moved into the local sync folder
+        Given user "Alice" has set up a client with default settings
+        And user "Alice" has created a folder "Folder1" inside the sync folder
+        And user "Alice" has created a file "newfile.txt" with size "50MB" in the temp folder
+        When user "Alice" moves file "newfile.txt" from the temp folder to "Folder1" in the sync folder
+        And the user waits for file "Folder1/newfile.txt" to be synced
```

Comment:
I think it will not make any real difference whether we create the file inside the sync folder or inside the temp folder and then move it.
So we can  just simply create it directly in the sync folder and that will save us the 2 new steps and reduce complexity.
Also the whole feature is about moving files / folders within the sync folder. Moving files from outside of the sync folder to inside is basically same as creating them freshly